src/libraries/System.Linq/src/System/Linq/Lookup.cs

```diff
@@ -76,7 +76,7 @@ public partial class Lookup<TKey, TElement> : ILookup<TKey, TElement>
     {
         private readonly IEqualityComparer<TKey> _comparer;
         private Grouping<TKey, TElement>[] _groupings;
-        private protected Grouping<TKey, TElement>? _lastGrouping;
+        internal Grouping<TKey, TElement>? _lastGrouping;
```

Comment:
What's referencing this?

Reply:
A bunch of the methods in Grouping, e.g. GroupByIterator.MoveNext